react/components/Spinner/__tests__/index.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import snapshotDiff from 'snapshot-diff'
+
+import Spinner from '../index'
+
+describe('Spinner spec', () => {
```

Comment:
```suggestion
describe('Spinner', () => {
```

Shouldn't we follow a pattern for the names of the tests?

Reply:
done